tests/Doctrine/Tests/ORM/Functional/Ticket/DDC933Test.php

```diff
@@ -29,7 +30,9 @@ public function testLockCTIClass()
         $this->_em->flush();
 
         $this->_em->beginTransaction();
-        $this->_em->lock($manager, \Doctrine\DBAL\LockMode::PESSIMISTIC_READ);
+        $this->_em->lock($manager, LockMode::PESSIMISTIC_READ);
         $this->_em->rollback();
+
+        $this->addToAssertionCount(1);
```

Comment:
The lock should have been released here - should we maybe just attempt a read? Since this is pessimistic mode, the query will succeed if the lock is gone, and it will fail or time-out if the lock wasn't released.

A second DB connection is needed to do that, though...